src/standalone/Vegetation/Canopy.jl

```diff
@@ -554,71 +532,25 @@ function ClimaLand.make_update_aux(
                     ),
                 ) * PlantHydraulics.harmonic_mean(areaip1, areai)
         end
-        # We update the fa[n_stem+n_leaf] element once we have computed transpiration, below
-        # update photosynthesis and conductance terms
-        # This should still use T_air, P_air, q_air
-        medlyn_factor =
-            @. lazy(medlyn_term(g1, T_air, P_air, q_air, thermo_params))
-        # Anywhere we use an Arrhenius factor, use T_canopy instead T_air
-        T_canopy = canopy_temperature(canopy.energy, canopy, Y, p, t)
-
-        # update moisture stress
-        i_end = n_stem + n_leaf
-        β = @. lazy(moisture_stress(ψ.:($$i_end) * ρ_l * grav, sc, pc))
-
-        # Update Rd, An, Vcmax25 (if applicable to model) in place
-        Vcmax25 = p.canopy.photosynthesis.Vcmax25
-        update_photosynthesis!(
-            Rd,
-            An,
-            Vcmax25,
-            canopy.photosynthesis,
-            T_canopy,
-            p.canopy.radiative_transfer.par.abs,
-            β,
-            medlyn_factor,
-            c_co2_air,
-            R,
-            energy_per_mole_photon_par,
-            par_d,
-        )
-        # update SIF
-        SIF = p.canopy.sif.SIF
-        update_SIF!(
-            SIF,
-            canopy.sif,
-            p.canopy.radiative_transfer.par.abs,
-            T_canopy,
-            Vcmax25,
-            R,
-            T_freeze,
-            canopy.photosynthesis.parameters,
-            energy_per_mole_photon_par,
-            par_d,
-        )
+        # We update the fa[n_stem+n_leaf] element once we have computed transpiration
+
+        # Update Rd, An, Vcmax25 (if applicable to model) in place, GPP
+        update_photosynthesis!(p, Y, t, canopy.photosynthesis, canopy)
         @. GPP = compute_GPP(An, extinction_coeff(G_Function, cosθs), LAI, Ω)
```

Comment:
should we update GPP inside of update_photosynthesis! too?

Reply:
good idea!